spec/lib/efile/nj/nj1040_spec.rb

```diff
@@ -104,6 +104,19 @@ def over_65_birth_year
     end
   end
 
+  describe 'line 13 - total exemptions' do
+    let(:intake) { create(:state_file_nj_intake) }
+    it 'sets line 13 to the sum of lines 6-12' do
+      self_exemption = 1000
+      expect(instance.lines[:NJ1040_LINE_6].value).to eq(self_exemption)
+      not_over_65 = 0
+      expect(instance.lines[:NJ1040_LINE_7].value).to eq(not_over_65)
+      not_blind = 0
+      expect(instance.lines[:NJ1040_LINE_8].value).to eq(not_blind)
+      expect(instance.lines[:NJ1040_LINE_13].value).to eq(self_exemption + not_over_65 + not_blind)
```

Comment:
[pebble] Could we adjust the input for this test to have values in all three fields that get summed? Right now, since two of them are zero, I could go and change the calculate_line_13 method to just sum the value of line 6 alone, and the test would still pass.

Reply:
Specifically I would recommend it just for this test - no need to do this change for the XML and PDF tests since they are not actually testing the summing behavior.

Reply:
That makes sense, good catch.